packages/inter-protocol/src/vaultFactory/burn.js

```diff
@@ -1,7 +1,7 @@
 // @jessie-check
 
 // XXX ambient types runtime imports until https://github.com/Agoric/agoric-sdk/issues/6512
-import '@agoric/zoe/exported.js';
```

Comment:
prune the XXX comment too?

Reply:
I'll do that. Any other requested changes? I'm putting this in to R4R since all the tests are passing. (I don't know what's up with the Cloudflare build but it works for me locally)